prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -659,10 +673,8 @@ def simple_retriver(  # type:ignore
         f" help extract relevant documents from a vector store:\n\n{prompt}"
     )
     answer = run_completion(
-        llm_helper=llm_helper,
-        llm_li=llm_li,
-        prompt=subq_prompt,
-        adapter_instance_id=output[PSKeys.LLM],
+        llm,
+        subq_prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
Done